app/src/main/res/values-de-rDE/strings.xml

```diff
@@ -0,0 +1,185 @@
+<resources>
+    <string name="app_name" translatable="false">PeakAR</string>
+
+    <!-- Google -->
+    <!-- Sign-in status messages -->
+    <string name="mountain_info" translatable="false">Mont Blanc - Monte Bianco lat: 45.8325 lon: 6.8641666666667 alt: 4810 <br /><br />
+                                 Dent du Geant lat:45.86355980599387 lon:6.951348205683087 alt: 4013<br /><br />
+                                 Aiguille du plan lat:45.891667 lon:6.907222 alt: 3673<br /><br />
+                                 Pointe de Lapaz lat:45.920774986207014 lon:6.812914656881065 alt: 3660</string>
+    <!-- Sign-in button labels and other strings -->
+
+    <string name="title_activity_main_menu" translatable="false">MainMenuActivity</string>
```

Comment:
Cirrus complains that this is translated in German. Since it is set to false it shouldn't